### PR TITLE
Don't create an operation out of path parameters (with unit test)

### PR DESCRIPTION
**Pull request by analogue**
Builds on PR #56 by adding a unit test.

### CODE REVIEW

File: bravado_core/resource.py [hunk 1]
```diff
@@ -49,6 +49,9 @@ def build_resources(swagger_spec):
     paths = swagger_spec.spec_dict['paths']
     for path_name, path_spec in iteritems(paths):
         for http_method, operation_spec in iteritems(path_spec):
+            if http_method == 'parameters':
```
Comment (prat0318) [inline bravado_core/resource.py]: It took me a while to understand how `parameters` can get assigned as `http_method`. A comment about this - handling `parameters` key in swagger spec which can be at same level as http methods - can be helpful later.

Comment (analogue) [inline bravado_core/resource.py]: Good point. I'll add something.

**Comment by prat0318** [comment]
flake8 complaining :(

**Comment by prat0318** [comment]
fix n ship

**Pull request merged**